elife/config/lib-systemd-system-uwsgi-service.service

```diff
@@ -14,5 +16,5 @@ Environment="LANG=en_US.UTF-8"
 Environment="NEW_RELIC_CONFIG_FILE={{ folder }}/newrelic.ini"
 ExecStart={{ folder }}/venv/bin/newrelic-admin run-program {{ folder }}/venv/bin/uwsgi --enable-threads --ini {{ folder }}/uwsgi.ini
 {% else %}
-ExecStart={{ folder }}/venv/bin/uwsgi --ini {{ folder }}/uwsgi.ini
+ExecStart={{ folder }}/venv/bin/uwsgi --ini {{ folder }}/uwsgi.ini --logto /var/log/uwsgi.log
```

Comment:
`logto` should be in `uwsgi.ini` right?

Reply:
it was in uwsgi.ini, I moved it into the service definition. It's something that shouldn't be configured on a per-app basis and the ownership of the log file changes between 14.04 and 16.04 too.